src/Microsoft.OData.Edm/Schema/EdmModel.cs

```diff
@@ -119,7 +119,7 @@ public void AddElements(IEnumerable<IEdmSchemaElement> newElements)
         /// Adds a vocabulary annotation to this model.
         /// </summary>
         /// <param name="annotation">The annotation to be added.</param>
-        public void AddVocabularyAnnotation(IEdmVocabularyAnnotation annotation)

```

Comment:
Hi, FYI, ODL has the extension method to set the location for annotation.

public static void SetSerializationLocation(this IEdmVocabularyAnnotation annotation, IEdmModel model, EdmVocabularyAnnotationSerializationLocation? location)

If we think add a parameter for "AddVocabularyAnnotation", we can add an extension method to "EdmModel" as `public void AddVocabularyAnnotation(this EdmModel model, IEdmVocabularyAnnotation annotation, Csdl.EdmVocabularyAnnotationSerializationLocation location)`

Reply:
I understand that the extension method is there. It is actually used in the implementation. 

Woudl you prefer the extension method? For the caller there is actually no difference due to the fact that the location parameter is optional.

Reply:
moved to extension method